frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/state.ts

```diff
@@ -18,7 +18,7 @@ export const initialState: State = {
   nodeNames: [], // nodes selected on the LVS step
   minDiskSize: '0',
   maxDiskSize: '',
-  diskSizeUnit: 'TiB',
+  diskSizeUnit: 'Ti',
```

Comment:
We should be showing the user in TiB? We need to make appropriate changes in the UI as well.

Reply:
It should be `Ti` only. On UI it is shwon as `TiB` only as it is mapped to enum.

Reply:
Okay so either previously this was a bug or now you're creating a bug as I don't see changes in the UI in this commit.

Reply:
yes, I missed to change it at this place. Hence, changing it here now.